oop/music/music.py

```diff
@@ -0,0 +1,22 @@
+
+class Music:
+    """A class that defines music"""
+
+    # defining that all music has a title, a bpm, and instruments
```

Comment:
good comments - they could have all been included in the docstring comment above (""")